greedybear/settings.py

```diff
@@ -15,6 +15,13 @@
 # SECURITY WARNING: don't run with debug turned on in production!
 DEBUG = os.environ.get("DEBUG", False) == "True"
 
+# DB Backup Config
+# Backup Interval in minutes. Default: 1440 min = 1 day.
+DB_BACKUP_FREQUENCY = int(os.getenv("DB_BACKUP_FREQUENCY", 1440))
```

Comment:
it is enough to allow users to just choose to do the download every X days. Then we set static the hour/minute time

Reply:
Ok Sir. I'll add that